### ISSUE TITLE
Get bitmap from screen data

### ISSUE THREAD

**Issue by raffahacks** [olikraus/u8glib#355]
I don't think it would be very difficult to code, but it would allow MANY more features, like for example animations that require the screen bitmap (am I clear in my explanation?)

**Comment by olikraus** [comment]
You simply can not get a bitmap from the current screen data. This is not possible with most of the LCDs.

**Comment by raffahacks** [comment]
Hello Olikraus,
Thanks for your answer. Sorry if I wasn't clear. What I mean is not getting
the screen bitmap from the screen directly, but instead telling u8glib to
store it in a variable (like XBMs) in the RAM while writing it on the
screen. An example:

u8g.firstPage();
u8g.displayToBitmap(OutputVariableNameHere);

do {
  // something
} while(u8g.nextPage());

Cheers

Il 28/lug/2015 23:52, "olikraus" <notifications@github.com> ha scritto:
>
> You simply can not get a bitmap from the current screen data. This is not
possible with most of the LCDs.

**Comment by olikraus** [comment]
ok, you want something like a virtual display, which outputs to RAM instead of the actual display.

**Comment by raffahacks** [comment]
Yes, something like that. I think many people would use it, but, as I am
not an expert C programmer, I don't know how easy it would be to code...
